src/controllers/project-controller.js

```diff
@@ -159,3 +162,108 @@ export async function getDuplicationRequestCriteria(req, res) {
 
   res.status(200).json({ oneTimeMedia, projectPublished, hasAccess })
 }
+
+export async function getPartitionSummary(req, res) {
+  const projectId = req.params.projectId
+  const partitionId = req.params.partitionId
+
+  const partition = await models.Partition.findByPk(partitionId)
+  let onetimeMedia = []
+  let views = 0
+  let specimens = 0
+  let labels = 0
+
+  const characters = await partitionService.getCharactersInPartitions(
+    partitionId
+  )
+  const taxa = await partitionService.getTaxaInPartitions(partitionId)
+
+  const taxaIds = Array.from(taxa.values())
+  const characterIds = Array.from(characters.values())
+
+  const taxaMedia =
+    taxaIds.length != 0 ? await mediaService.getMediaByTaxaIds(taxaIds) : []
+  const characterMedia =
+    characterIds.length != 0
+      ? await mediaService.getMediaByCharacterIds(characterIds)
+      : []
+
+  const connectedMediaIds = taxaMedia.filter((media) => {
+    return !characterMedia.includes(media)
+  })
+
+  if (connectedMediaIds.length > 0) {
+    const mediaIds = connectedMediaIds.map((media) => media.media_id)
+
+    const medias = await models.MediaFile.findAll({
+      where: { media_id: mediaIds },
+    })
+
+    onetimeMedia = medias.filter(
+      (media) => media.is_copyrighted > 0 && media.copyright_license == 8
+    )
+    views = medias.map((media) => media.views).length
+    specimens = medias.map((media) => media.specimens).length
+    labels = mediaService.getMediaLabels(mediaIds).length
+  }
+
+  const documents = documentService.getDocuments(projectId).length
+  const bibliographicReferences =
+    bibliographyService.getBibliographiesByProjectId(projectId).length
```

Comment:
Same as above. This count uses projectId whereas the original one used partitionId.